src/main/java/seedu/duke/manager/OrderManager.java

```diff
@@ -5,42 +5,66 @@
 
 import java.util.ArrayList;
 
-
+/**
+ * OrderManager.
+ */
 public class OrderManager {
-    private final ArrayList<Order> orders;
-    private final DishManager dishManager = new DishManager();
-    private final ArrayList<Dish> dishes;
+    private ArrayList<Order> orders;
+    private DishManager dishManager = new DishManager();
```

Comment:
Same as above, should use the same `DishManager` instance.

Reply:
Thanks

Reply:
if controllers depend on another manager, we may need to move the manager instantiation into the maincontroller.

Reply:
or, we can use factory classes to always get the same instance of the manager in different controllers.

Reply:
prefer the second